howso/visuals/visuals.py

```diff
@@ -581,3 +581,81 @@ def plot_interpretable_prediction(
 
     if len(figures) == 1:
         return figures[0]
+
+
+def plot_fairness_disparity(
+    fairness_results: dict,
+    reference_class: str,
+    *,
+    fairness_threshold: float = 0.75,
+    x_tickangle: bool | float = False,
+    fair_color: str = "#9BBf85",
+    unfair_color: str = "#B3589A",
+    reference_class_color: str = 'grey',
+):
+    """
+    Helper function for plotting fairness disparity results.
+
+    Parameters
+    ----------
+    fairness_results : dict
+        Dictionary of the fairness disparity ratios.
+    reference_class : str
+        The reference class for the metric calculation.
+    fairness_threshold : float, default 0.75
+        Threshold for values to be classified as fair. Values below this threshold are colored red when
+        graphing while values above are colored green.
+    x_tickangle : bool | float, default False
+        The angle of rotation for the x-axis labels. If False, the labels will not be rotated.
+    fair_color : str, default '#9BBf85'
+        The bar chart color for values that are above the ``fairness_threshold``. Accepts `Plotly` compatible
+        values.
+    unfair_color : str, default '#B3589A'
+        The bar chart color for values that are below the ``fairness_threshold``. Accepts `Plotly` compatible
+        values.
+    reference_class_color : str, default 'grey'
+        The bar chart color for the reference class. Accepts `Plotly` compatible
+        values.
+
+    Returns
+    -------
+    Figure
+        The resultant `Plotly` figure.
+
+    Examples
+    --------
+    .. code-block::python
+
+        >> fairness_results
+            {
+                'Dataset1': {'Male': 0.8, 'Female': 0.7, 'Other': 0.9},
+                'Dataset2': {'Male': 0.6, 'Female': 0.9, 'Other': 0.5},
+                'Dataset3': {'Male': 0.7, 'Female': 0.4, 'Other': 0.6}
+            }
+        >> fig = plot_fairness_disparity(fairness_results, reference_class='Male')
+
+    """
+    num_subplots = len(fairness_results)
+    fig = make_subplots(rows=1, cols=num_subplots, subplot_titles=list(fairness_results.keys()))
+
+    for i, (_, results) in enumerate(fairness_results.items(), 1):
+        sorted_data = {key: results[key] for key in sorted(results, key=lambda x: (x != reference_class, x))}
+        for key, value in sorted_data.items():
+            color = reference_class_color if key == reference_class else (
+                unfair_color if value < fairness_threshold else fair_color
+            )
+            key_value = key if key != reference_class else f'{reference_class} (ref)'
+
+            fig.add_trace(
+                go.Bar(x=[key_value], y=[value], marker_color=color, name=key_value),
+                row=1, col=i
+            )
+            fig.add_annotation(x=key_value, y=value / 2, text=str(round(value, 2)), showarrow=False,
+                               xref=f"x{i}", yref=f"y{i}")
+
+    fig.update_layout(showlegend=False)
+
+    if x_tickangle:
```

Comment:
I think this probably needs to be a bit more complicated to account for the new `bool` potential of the parameter.